lib/pubsub/subscription.js

```diff
@@ -371,4 +374,35 @@ Subscription.prototype.pull = function(options, callback) {
   });
 };
 
+/**
+ * Modify the ack deadline for a specific message. This method is useful to
+ * indicate that more time is needed to process a message by the subscriber, or
+ * to make the message available for redelivery if the processing was
+ * interrupted.
+ *
+ * @param {object}   options - The configuration object.
+ * @param {number}   options.ackId - The ack id to change.
+ * @param {number}   options.seconds - Number of seconds after call is made to
+ *     set the deadline of the ack.
+ * @param {Function=} callback - The callback function.
```

Comment:
Formatting here is inconsistent with other areas of the docs. To match, this would be (note the spacing and casing):

```js
@param {type} label - Description.
```